server/src/main/java/com/google/speech/tools/voxetta/services/PromptService.java

```diff
@@ -36,4 +38,20 @@
      * @return one prompt from the Prompt database.
      */
     public String getOnePrompt();
+
+    /**
+     * Retrieves all prompts in the database as Entities. Primarily for manual debugging.
+     *
+     * @return {String} JSON array of all Prompt Entities.
+     */
+    public String getAllPrompts();
```

Comment:
I would prefer to have these debugging tools split out into like a `PromptDebugService` and `PromptDebugServlet` so they can easily be isolated and turned off